.evergreen/config.yml

```diff
@@ -417,9 +417,7 @@ axes:
     display_name: Driver Version
     values:
       - id: "latest-stable"
-        display_name: "1.8.0beta2"
-        variables:
-          DRIVER_VERSION: "1.8.0beta2"
+        display_name: "1.8-stable"
```

Comment:
This was left over from earlier tests where we didn't have 1.8 stable yet. We can now omit the `DRIVER_VERSION` constant to always install the current stable release.